packages/agents/router/src/errors/execute.ts

```diff
@@ -52,50 +64,56 @@ export class NotEnoughLiquidity extends NxtpError {
   }
 }
 
-export class NotEnoughAmount extends NxtpError {
+export class NotEnoughAmount extends ExecuteError {
   constructor(context: any = {}) {
     super("Not enough tokens for swap", context, NotEnoughAmount.name);
   }
 }
 
-export class CallDataForNonContract extends NxtpError {
+export class CallDataForNonContract extends ExecuteError {
   constructor(context: any = {}) {
     super("Calldata specified for an address that is not a contract", context, CallDataForNonContract.name);
   }
 }
 
-export class RouterNotApproved extends NxtpError {
+export class RouterNotApproved extends ExecuteError {
   constructor(context: any = {}) {
     super("Router not approved", context, RouterNotApproved.name);
   }
 }
 
-export class SequencerResponseInvalid extends NxtpError {
+export class SequencerResponseInvalid extends ExecuteError {
   constructor(context: any = {}) {
     super("sendBid: POST request returned invalid response", context, SequencerResponseInvalid.name);
   }
 }
 
-export class AuctionExpired extends NxtpError {
+export class AuctionExpired extends ExecuteError {
   constructor(context: any = {}) {
     super("Auction has already expired for this transfer.", context, AuctionExpired.name);
   }
 }
 
-export class SanityCheckFailed extends NxtpError {
+export class SanityCheckFailed extends ExecuteError {
   constructor(context: any = {}) {
     super("Sanity check failed", context, SanityCheckFailed.name);
   }
 }
 
-export class NomadHomeBlacklisted extends NxtpError {
+export class NomadHomeBlacklisted extends ExecuteError {
   constructor(context: any = {}) {
     super("Home network is on blacklist (nomad down)", context, NomadHomeBlacklisted.name);
   }
 }
 
-export class InvalidAuctionRound extends NxtpError {
+export class InvalidAuctionRound extends ExecuteError {
   constructor(context: any = {}) {
     super("Invalid auction round", context, InvalidAuctionRound.name);
   }
 }
+
+export class UnableToGetAsset extends ExecuteError {
```

Comment:
Is this a `retryable = true` error in all cases ?

Reply:
It's retryable = false.